sdk_container/src/third_party/coreos-overlay/dev-lang/rust/rust-1.79.0.ebuild

```diff
@@ -387,8 +388,8 @@ src_configure() {
 		enable-warnings = false
 		[llvm.build-config]
 		CMAKE_VERBOSE_MAKEFILE = "ON"
-		CMAKE_C_FLAGS_${cm_btype} = "${CFLAGS}"
-		CMAKE_CXX_FLAGS_${cm_btype} = "${CXXFLAGS}"
+		CMAKE_C_FLAGS_${cm_btype} = "-O2 -pipe"
```

Comment:
these have to stay until rust gets properly built using the portage-stable build file, to avoid these hacks

Reply:
CXXFLAGS and CFLAGS inherit the mtune=generic flag, which is not supported on the cross building for riscv64 arch